src/routes/index.jsx

```diff
@@ -19,7 +19,12 @@ const Routes = () => {
                 <PublicRoute exact path = "/restaurar-contrasena/:token/" component = { ChangePassword } />
 
                 {/* Private Routes */}
-                <PrivateRoute exact path = '/dashboard/' component = { Dashboard } />
```

Comment:
Yo don't have any private route now, if you're not using this import, comment it to avoid eslint errors.